src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/ListWithDefaultNullFW.java

```diff
@@ -0,0 +1,394 @@
+        int fieldLimit = offset + FIRST_FIELD_OFFSET;
+        bitmask = 0;
+        final int length = length();
+        for (int field = INDEX_VARIANT_OF_STRING1; field < length; field++)
+        {
+            switch (field)
+            {
+            case INDEX_VARIANT_OF_STRING1:
+                variantOfString1RO.wrap(buffer, fieldLimit, maxLimit);
+                fieldLimit = variantOfString1RO.limit();
+                bitmask |= 1 << INDEX_VARIANT_OF_STRING1;
+                break;
+            case INDEX_VARIANT_OF_STRING2:
+                if (buffer().getByte(fieldLimit) != NULL_VALUE)
+                {
+                    variantOfString2RO.wrap(buffer, fieldLimit, maxLimit);
+                    fieldLimit = variantOfString2RO.limit();
+                    bitmask |= 1 << INDEX_VARIANT_OF_STRING2;
+                }
+                else
+                {
+                    fieldLimit += NULL_VALUE_SIZE;
+                }
+                break;
+            case INDEX_VARIANT_OF_UINT:
+                if (buffer().getByte(fieldLimit) != NULL_VALUE)
+                {
+                    variantOfUintRO.wrap(buffer, fieldLimit, maxLimit);
+                    fieldLimit = variantOfUintRO.limit();
+                    bitmask |= 1 << INDEX_VARIANT_OF_UINT;
+                }
+                else
+                {
+                    fieldLimit += NULL_VALUE_SIZE;
+                }
+                break;
+            case INDEX_VARIANT_OF_INT:
+                if (buffer().getByte(fieldLimit) != NULL_VALUE)
+                {
+                    variantOfIntRO.wrap(buffer, fieldLimit, maxLimit);
+                    fieldLimit = variantOfIntRO.limit();
+                    bitmask |= 1 << INDEX_VARIANT_OF_INT;
+                }
+                else
+                {
+                    fieldLimit += NULL_VALUE_SIZE;
+                }
+                break;
+            }
+        }
+        checkLimit(limit(), maxLimit);
+        return this;
+    }
+
+    @Override
+    public ListWithDefaultNullFW tryWrap(
+        DirectBuffer buffer,
+        int offset,
+        int maxLimit)
+    {
+        if (super.tryWrap(buffer, offset, maxLimit) == null)
+        {
+            return null;
+        }
+        int fieldLimit = offset + FIRST_FIELD_OFFSET;
+        bitmask = 0;
+        final int length = length();
+        for (int field = INDEX_VARIANT_OF_STRING1; field < length; field++)
+        {
+            switch (field)
+            {
+            case INDEX_VARIANT_OF_STRING1:
+                if (variantOfString1RO.tryWrap(buffer, fieldLimit, maxLimit) == null)
+                {
+                    return null;
+                }
+                fieldLimit = variantOfString1RO.limit();
+                bitmask |= 1 << INDEX_VARIANT_OF_STRING1;
+                break;
+            case INDEX_VARIANT_OF_STRING2:
+                if (buffer().getByte(fieldLimit) != NULL_VALUE)
```

Comment:
Make sure we can handle the case where bytes are insufficient to read potentially null value.